packages/opencensus-core/src/common/console-logger.ts

```diff
@@ -104,10 +101,6 @@ export class ConsoleLogger implements types.Logger {
   }
 }
 
-
-// TODO: reevaluate the need to create a new logger instance on every call to
```

Comment:
By creating a new logger instance on every call to it we allow distinct classes to have distinct log levels.

Reply:
I think the original comment that this was in reference to that every time `logger` was called, the field `logger['logger']` was being replaced. This field is also never getting read. The `logger` method is otherwise just a call-through to `new ConsoleLogger`.

I would prefer that `logger` not even be used in the first place, and that anywhere it's used, `new ConsoleLogger` is used instead. After all, why do we need the wrapper method?

Reply:
See: https://github.com/census-instrumentation/opencensus-node/pull/2#discussion_r185030617

Reply:
My mistake, I got the TODO confused. Regarding this one, our idea here is to return an instance of logger, although calling `new ConsoleLogger()` works too. We drove inspiration from [this logDriver](https://github.com/cainus/logdriver/blob/bba1761737ca72f04d6b445629848538d038484a/index.js#L50).

I removed the `logger['logger'] = aLogger;` that had no need to be there at all.